src/scenes/App/styles.scss

```
@@ -46,10 +47,13 @@
   }
 
   header {
-    margin-bottom: $space-extra-large2;
+    margin-bottom: $space-extra-large5;
   }
 
   section:not(:last-child) {
     margin-bottom: $space-extra-large6;
   }
+  .point {
+    margin: 0 0 $space-extra-large 0;
```

Inte helt samma sak men går det att ha bara margin-bottom här? 😊 
```suggestion
    margin-bottom: $space-extra-large;
```